packages/generator-sketch/CHANGELOG.md

```diff
@@ -62,3 +62,143 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 # [3.0.0-beta.2](https://github.com/telekom/scale/compare/v0.1.4...v3.0.0-beta.2) (2021-04-20)
 
 **Note:** Version bump only for package @telekom/scale-generator-sketch
+
```

Comment:
can you remove these additions which are before v3, please?

Reply:
absolutely, wasn't sure if we wanted to keep them^^

Reply:
but yeah, nothing meaningful in there 😅